packages/terra-worklist-data-grid/tests/jest/WorklistDataGrid.test.jsx

```diff
@@ -113,4 +113,18 @@ describe('WorklistDataGrid', () => {
 
     console.error.mockRestore();
   });
+
+  it('test blank cells', () => {
```

Comment:
This is too vague, can we update the test description to be more explicit, e.g.
(this can be replaced with a more accurate statement)
```suggestion
  it('creates a blank cell if the cell data is empty', () => {
```